log4j-mongodb/src/test/java/org/apache/logging/log4j/mongodb/MongoDbProviderTest.java

```diff
@@ -0,0 +1,108 @@
+package org.apache.logging.log4j.mongodb;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
```

Comment:
Please use the JUnit 5 `org.junit.jupiter.api.Assertions` class instead.

Reply:
Done